cosmos/providers/dbt/core/operators/local.py

```diff
@@ -66,9 +66,9 @@ def exception_handling(self, result: FullOutputSubprocessResult):
             raise AirflowSkipException(
                 f"dbt command returned exit code {self.skip_exit_code}. Skipping."
             )
-        elif result.exit_code != 0:
             raise AirflowException(
-                f"dbt command failed. The command returned a non-zero exit code {result.exit_code}."
+                f"dbt command failed. The command returned a non-zero exit code {result.exit_code}. Details: "
```

Comment:
Does the "SkipException" do some magic?  I'm wondering how we can raise 2 exceptions in a row like this.

Reply:
Well spotted, @joedborg ! I deleted the `elif` line by mistake, and we had no tests covering the other branch.  Thanks a lot for catching this.